packages/react-ui/src/stories/common/Text-field.stories.js

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import { Grid, InputAdornment, SvgIcon, TextField, Typography } from '@material-ui/core';
+
+export default {
+  title: 'Common/Text Field',
+  component: TextField,
+  argTypes: {
+    variant: {
+      control: {
+        type: 'select',
+        options: ['standard', 'filled', 'outlined']
+      }
+    },
+    required: {
+      control: {
+        type: 'boolean'
+      }
+    },
+    disabled: {
+      control: {
+        type: 'boolean'
+      }
+    },
+    label: {
+      control: {
+        type: 'text'
+      }
+    }
+  }
+}
+const AccountIcon = (props) => (
+  <SvgIcon {...props}>
+    <g fill="#4d4d4d">
+      <circle cx="10" cy="10" fill="transparent" strokeWidth="2px" stroke="#4d4d4d" r="6"/>
+      <circle cx="10" cy="10" r="2"/>
+    </g>
+  </SvgIcon>
+);
+const EyeIcon = (props) => (
+  <SvgIcon {...props}>
+    <g transform="matrix(.02146 0 0 .02146 1 1)" fill="#4d4d4d">
+      <path d="m466.07 161.53c-205.6 0-382.8 121.2-464.2 296.1-2.5 5.3-2.5 11.5 0 16.9 81.4 174.9 258.6 296.1 464.2 296.1 205.6 0 382.8-121.2 464.2-296.1 2.5-5.3 2.5-11.5 0-16.9-81.4-174.9-258.6-296.1-464.2-296.1m0 514.7c-116.1 0-210.1-94.1-210.1-210.1 0-116.1 94.1-210.1 210.1-210.1 116.1 0 210.1 94.1 210.1 210.1 0 116-94.1 210.1-210.1 210.1"/>
+      <circle cx="466.08" cy="466.02" r="134.5"/>
+    </g>
+  </SvgIcon>
+);
```

Comment:
Use original Material-UI icons or make these icons to inherit the fill color from `SvgIcon`, as icons in TextFields and other Material-UI components are tintable.
Check https://github.com/CartoDB/airship/blob/82033af527913ae6ba5dff2efca2d0ebf74e86bd/packages/react-ui/src/widgets/WrapperWidgetUI.js#L84 for the latter option.